lametro/templates/lametro/legislation.html

```diff
@@ -97,20 +98,19 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                             </tr>
                         </thead>
                         <tbody>
-                            {% for action in actions %}
+                            {% for action in events %}
                                 <tr>
                                     <td class='nowrap text-muted small'>
-                                        <span datetime='{{action.date_dt | format_date_sort }}'>{{action.date_dt|date:'n/d/y'}}</span>
+                                        <span datetime='{{action.date}}'>{{action.date}}</span>
                                     </td>
                                     <td class="small">
-                                        <span class='text-{{action.label}}'>{{action.description | remove_action_subj}}</span>
-                                        {% if action.related_organization %}
-                                            to
-                                            {{action.related_organization.link_html|safe}}
-                                        {% endif %}
+                                        <span class='text-default'>{{action.description}}</span>
                                     </td>
                                     <td class="small">
-                                        {{action.organization.link_html|safe}}
+                                            {{ action.organization.link_html|safe }}
+                                        {% if action.description == 'CANCELLED' %}
```

Comment:
`CANCELLED` is an event status, so you should check that, rather than the action description, to determine whether to include the `<strike>` element and label. It also looks like you're missing the opening strike element. You're looking for something like:

```html
{% if the meeting is cancelled %}<strike>{% endif %}
Organization link
{% if the meeting is cancelled %}
</strike>
<!-- label -->
{% endif %}
```

See Jasmine's commit: https://github.com/datamade/la-metro-councilmatic/pull/450/commits/7de05bfbec2588432d8b2cdcd906a459b1cdad09#diff-73a27e2aba0b040dc38e92e1f11e8b87R114